.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "diffrential-privacy"]
+	path = diffrential-privacy
```

Comment:
careful you have a small typo in your submodule path "diffrential-privacy" -> "diff**e**rential-privacy"